### PR TITLE
Add a 'Copy' button to the fax UI

### PR DESCRIPTION
**Pull request by arimah**
## About the PR
Adds a 'Copy' button to the fax UI, which causes the inserted paper to be copied immediately.

## Why / Balance
Sometimes we make paperwork - cargo order forms, crew transfer forms, various other documentation that is worth keeping (and adds gameplay to HoP and QM!). Copying such papers is traditionally a tedious job of either copy-pasting onto blank sheets and hoping desperately that you have enough blank papers lying around, or, when you run out of paper, finding a fax machine, sending it somewhere else, and running halfway across the station to pick it up. With the 'Copy' button, you can do it all without even leaving your desk!

## Technical details
Content.Server.FaxSystem has a few new methods, most notably `Copy()`. I chose to add a new method that duplicates some functionality because copying does not require use of device networking, and also should not play the SendSound to avoid clashing with the printing sound (which starts immediately).

## Media
https://github.com/space-wizards/space-station-14/assets/30327355/baccdd6b-6f99-4511-a03a-6bad256740f9

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
None.

**Changelog**
:cl:
- add: Faxes now have a Copy button for copying the inserted paper.

### CODE REVIEW

**Comment by arimah** [comment]
Possible alternative UI:

image

**Comment by VektorZ1** [comment]
Finally

**Comment by arimah** [comment]
@dvir001 Huh, I had no idea Frontier had this feature! Very similar code, too. Mine is entirely original work, despite the similarities. One major difference is my `Copy()` method completely bypasses device networking – it just enqueues the printout directly. Not sure which is better!

File: Content.Server/Fax/FaxSystem.cs [hunk 1]
```diff
@@ -43,6 +43,12 @@ public sealed class FaxSystem : EntitySystem
 
     private const string PaperSlotId = "Paper";
 
+    /// <summary>
+    ///     The prototype ID to use for faxed or copied entities if we can't get one from
+    ///     the paper entity for whatever reason.
+    /// </summary>
+    private const string DefaultPaperPrototypeId = "Paper";
```
Comment (mirrorcult) [inline Content.Server/Fax/FaxSystem.cs:50]:
```suggestion
    /// <summary>
    ///     The prototype ID to use for faxed or copied entities if we can't get one from
    ///     the paper entity for whatever reason.
    /// </summary>
    [ValidatePrototypeId<EntityPrototype>]
    private const string DefaultPaperPrototypeId = "Paper";
```

**Review by mirrorcult — CHANGES_REQUESTED** [review]
good to go outside of nitpick

File: Content.Server/Fax/FaxSystem.cs (hunk 1 above)
Comment (arimah) [reply Content.Server/Fax/FaxSystem.cs]: Aha! What a handy attribute – I shall have to remember that.

**Comment by K-Dynamic** [comment]
Let the boykisser abuse start

**Pull request merged**